geode-core/src/integrationTest/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImplJUnitTest.java

```diff
@@ -420,6 +420,56 @@ public void testLocatorUpdateIntervalZero() {
     assertEquals(updateLocatorInterval, source.getLocatorUpdateInterval());
   }
 
+  /**
+   * This tests validates update of IP of locator from initialLocators list.
+   * In this test we validate this using identityHashCode.
+   */
+  @Test
+  public void testinitialLocatorIpChange() {
+    int port = 11011;
+    List<InetSocketAddress> locators = new ArrayList<>();
+    InetSocketAddress floc1 = InetSocketAddress.createUnresolved("localhost", port);
```

Comment:
`floc1` and `floc2` were confusing names to me (mostly because by the end of the test I forgot what they stood for and started thinking about a flock of birds, but I digress...)

I think some thing like `unresolvedLocalhost` and `fakeLocalhost` would be a bit more clearer